resizer.go

```diff
@@ -68,8 +69,9 @@ func resizer(buf []byte, o Options) ([]byte, error) {
 	}
 
 	// Try to use libjpeg shrink-on-load
-	if imageType == JPEG && shrink >= 2 {
-		tmpImage, factor, err := shrinkJpegImage(buf, image, factor, shrink)
+	supportsShrinkOnLoad := imageType == JPEG || imageType == WEBP && VipsMajorVersion >= 8 && VipsMinorVersion >= 3
```

Comment:
Should not the `libvips` runtime version condition only be a constraint for webP images?

Reply:
We are still supporting 7.42+ and 8.0+ `libvips` versions:
http://www.vips.ecs.soton.ac.uk/supported/7.42/doc/html/libvips/VipsForeignSave.html#vips-jpegload

Reply:
libwebp does any kind of shrink, I can change that.
https://github.com/jcupitt/libvips/blob/5c6979cdcbd1dd81edfe237fdf7d0355e84c7b54/libvips/foreign/webp2vips.c#L194-L207

Reply:
@h2non `&&` has the priority over `||` iirc.

```jpg + webp * major * minor```